counsel.el

```diff
@@ -191,11 +191,9 @@ Update the minibuffer with the amount of lines collected every
       (with-current-buffer (process-buffer process)
         (goto-char (point-min))
         (setq size (- (buffer-size) (forward-line (buffer-size))))
-        (ivy--set-candidates
-         (split-string
-          (buffer-string)
-          counsel-async-split-string-re
-          t)))
+        (let ((cands (split-string (buffer-string) counsel-async-split-string-re t)))
+          (setq ivy--old-cands ivy--all-candidates)
```

Comment:
I thought the "killed" message was for any signal, but according to the [documentation](https://www.gnu.org/software/emacs/manual/html_node/elisp/Sentinels.html), that looks like its just for SIGKILL, which is what [delete-process](https://www.gnu.org/software/emacs/manual/html_node/elisp/Deleting-Processes.html) sends.